docs/page-objects/rules.rst

```diff
@@ -390,6 +390,58 @@ in the :class:`~.ApplyRule` to be written into ``web_poet.default_registry``.
 
     The next section explores this caveat further.
 
+Using URLs against the registered rules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One of the important aspects of :class:`~.ApplyRule` is dictating which URLs it's
+able to work using its ``for_patterns`` attribute. There are a few methods
+available in :class:`~.RulesRegistry` which accepts a URL value (:class:`str`,
+:class:`~.RequestUrl`, or :class:`~.ResponseUrl`) to find specific information
+from the registered rules.
+
+.. _rules-overrides_for-example:
+
+Find the page object overrides
+""""""""""""""""""""""""""""""
+
+Suppose you want to see what are the :ref:`rules-intro-overrides` that are
+available from a given webpage, you can use :meth:`~.RulesRegistry.overrides_for`
+by passing the webpage URL. For example:
+
+.. code-block:: python
+
+    from web_poet import default_registry
+
+    overrides = default_registry.overrides_for("http://books.toscrape.com/")
+    print(overrides)
+
+    # {
+    #     <class 'OldProductPage'>: <class 'NewProductPage'>,
+    #     <class 'OverriddenPage'>: <class 'UseThisPage'>,
+    # }
+
+It returns a :class:`Mapping` where the *key* represents the page object class
+that is overridden or replaced by the page object class in the *value*.
+
+.. _rules-page_object_for_item-example:
+
+Identify the page object that could create the item
+"""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Suppose you want to retrieve the page object class that is able to create the
+item class that you want from a given webpage, you can use
+:meth:`~.RulesRegistry.page_object_for_item`. For example:
+
+.. code-block:: python
+
+    from web_poet import default_registry
+
+    item_cls = default_registry.page_object_for_item(
```

Comment:
shouldn't it be page_cls?

Reply:
Do you mean renaming it something like `page_object_cls_for_item`? or maybe `page_cls_for_item`?

Reply:
The varible is named `item_cls`, but it contains page class, not item class.

Reply:
I totally missed that. Thanks! Updated. https://github.com/scrapinghub/web-poet/pull/112/commits/1ab1a43abf2a3dc4c490216ebb6bc301bec6522e

Reply:
You actually raised a good point - should we rename this method to page_cls_for_item, as it doesn't return an instantiated object?

Reply:
Yeah, I think that's much better. https://github.com/scrapinghub/web-poet/pull/112/commits/e86bf379f8ca1ddd9c6479499a1988e1b47b794a